Makefile.include

```diff
@@ -186,13 +186,11 @@ endif
 
 # Fail on warnings. Can be overridden by `make WERROR=0`.
 WERROR ?= 1
-export WERROR
```

Comment:
Add to `dist/tools/buildsystem_sanity_check/check.sh` EXPORTED_VARIABLES_ONLY_IN_VARS. Same below.